codefresh-tunnel-client/tests/values/required-values.yaml

```diff
@@ -0,0 +1,7 @@
+tunnelServer:
+    host: server
+    port: 443
```

Comment:
`port` should be optional (I know its only a test by just making sure)

Reply:
Yep, it is optional (cause it is set in default values.yaml to be 443.

Reply:
https://github.com/codefresh-io/codefresh-tunnel-charts/blob/20497831fae98924e398d528a38071b54762d72f/codefresh-tunnel-client/values.yaml#L12